### ISSUE TITLE
Write Unittests for `lib/views/after_auth_screens/profile/edit_profile_page.dart`

### ISSUE THREAD

**Issue by palisadoes** [PalisadoesFoundation/talawa#2621]
We will need unittests done for all methods, classes and functions found in this file.

Any widgets referenced in this file must also have unittests done

### PR Acceptance Criteria

1. When complete this file must show **100%** coverage when merged into the code base.
1. [The current code coverage for the file can be found here](https://app.codecov.io/gh/PalisadoesFoundation/talawa/tree/develop/?displayType=list)
1. CodeRabbit must approve all your changes to the test file(s) and the file(s) under test
1. The PR will show a report for the code coverage for the file you have added. You can use that as a guide
1. No functionality must be removed to achieve this goal
1. All tests must be valid and truly test the code base

**Potential internship candidates**

Please read this if you are planning to apply for a Palisadoes Foundation internship 
- https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by karthik11135** [comment]
Hi, Could you please assign this to me?

**Comment by xkaper001** [comment]
I noticed that this issue is unassigned, and I'd love to work on it!
Could you please assign this issue to me?

Thank you!

**Comment by palisadoes** [comment]
Are you still working on this?

**Comment by xkaper001** [comment]
Yes I'm working on this. I could get the file coverage to 99% but that 1% is what I'm stuck on.

**Comment by palisadoes** [comment]
Unassigning. Inactivity

**Comment by MayankJha014** [comment]
can you assign this issue to me @palisadoes

**Comment by atharvajagtap112** [comment]
can you assign the issue to me

**Comment by MayankJha014** [comment]
started working on it

**Comment by MayankJha014** [comment]
@palisadoes Sorry sir but unable to figure what i should add on this

**Comment by palisadoes** [comment]
https://app.codecov.io/gh/PalisadoesFoundation/talawa/blob/develop/lib%2Fviews%2Fafter_auth_screens%2Fprofile%2Fedit_profile_page.dart

**Comment by MayankJha014** [comment]
@palisadoes @noman2002 So Sir this issue is resolved
